packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/containers.html

```diff
@@ -2,6 +2,14 @@
     <cui-message-container
         data-messages="$ctrl.messages"
     ></cui-message-container>
+    <storages-components-user-credential-banner
+        ng-if="$ctrl.createdContainerInfos"
+        data-user="$ctrl.createdContainerInfos.user"
+        data-role="$ctrl.createdContainerInfos.role"
+        data-credential="$ctrl.createdContainerInfos.userCredentials"
+        data-container-name="$ctrl.createdContainerInfos.containerName"
+        data-field-secret-key-label="{{:: 'pci_projects_project_storages_cold_archives_add_container_success_secret_label' | translate}}"
+    ></storages-components-user-credential-banner>
```

Comment:
**nitpick:** Use prefix to avoid name collision
```suggestion
    <pci-projects-project-storages-components-user-credential-banner
        ng-if="$ctrl.createdContainerInfos"
        data-user="$ctrl.createdContainerInfos.user"
        data-role="$ctrl.createdContainerInfos.role"
        data-credential="$ctrl.createdContainerInfos.userCredentials"
        data-container-name="$ctrl.createdContainerInfos.containerName"
        data-field-secret-key-label="{{:: 'pci_projects_project_storages_cold_archives_add_container_success_secret_label' | translate}}"
    ></pci-projects-project-storages-components-user-credential-banner>
```